assets/stylesheets/shipping-label.scss

```diff
@@ -7,26 +7,54 @@
 .wcc-shipping-label-dialog__content {
 	display: flex;
 	flex-direction: column;
-	height: 100%;
+	max-height: 100%;
```

Comment:
With this style, the scroll bar can show even if the modal has enough height. See this (Chrome in Windows):
![scroll](https://cloud.githubusercontent.com/assets/1715800/18003239/61f9d614-6b84-11e6-88ea-2c4396929eb2.png)

Maybe in Mac it's not noticeable because of the invisible scrollbars :)
Putting back `height: 100%` here solves it (I don't know if it's some kind of CSS heresy though)